cmd/kops/create.go

```diff
@@ -136,7 +137,7 @@ func RunCreate(f *util.Factory, out io.Writer, c *CreateOptions) error {
 			}
 		}
 		// TODO: this does not support a JSON array
-		sections := bytes.Split(bytes.Replace(contents, []byte("\r\n"), []byte("\n"), -1), []byte("\n---\n"))
+		sections := text.SplitContentToSections(contents)
```

Comment:
This is the perfect name for the function - I was wondering "how did I now know about that function" as it looked so much like a k8s function :-)